Lib/test/test_dis.py

```diff
@@ -421,6 +422,133 @@ def _fstring(a, b, c, d):
            RETURN_VALUE
 """ % (_fstring.__code__.co_firstlineno, _fstring.__code__.co_firstlineno + 1)
 
+def _with(c):
```

Comment:
Why do we care what the exact output of the compiler is?
Would a simple test that counts the number of instructions to avoid accidental regressions, suffice?

Reply:
I'm not sure that counting instructions is much less fragile (if the compiler changes the number of instructions can easily change), but the test output is a lot less clear then in terms of what has changed.